CHANGELOG.md

```diff
@@ -1,3 +1,193 @@
+- Bump jest from 27.4.4 to 27.4.5 [#159](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/159) ([@dependabot](https://github.com/dependabot))
+- Bump eslint from 8.4.1 to 8.5.0 [#158](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/158) ([@dependabot](https://github.com/dependabot))
+- Bump @vercel/ncc from 0.33.0 to 0.33.1 [#157](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/157) ([@dependabot](https://github.com/dependabot))
+- Bump jest from 27.4.3 to 27.4.4 [#155](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/155) ([@dependabot](https://github.com/dependabot))
+- Bump eslint from 8.4.0 to 8.4.1 [#154](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/154) ([@dependabot](https://github.com/dependabot))
+- Bump eslint from 8.3.0 to 8.4.0 [#153](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/153) ([@dependabot](https://github.com/dependabot))
+- Bump jest from 27.3.1 to 27.4.3 [#152](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/152) ([@dependabot](https://github.com/dependabot))
+- Bump @vercel/ncc from 0.31.1 to 0.33.0 [#151](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/151) ([@dependabot](https://github.com/dependabot))
+- Bump prettier from 2.5.0 to 2.5.1 [#150](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/150) ([@dependabot](https://github.com/dependabot))
+- ci: enough of the devDependencies PR noise [#149](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/149) ([@consideRatio](https://github.com/consideRatio))
+- Bump prettier from 2.4.1 to 2.5.0 [#148](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/148) ([@dependabot](https://github.com/dependabot))
+- Bump eslint-plugin-jest from 25.2.4 to 25.3.0 [#147](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/147) ([@dependabot](https://github.com/dependabot))
+- Bump eslint from 8.2.0 to 8.3.0 [#146](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/146) ([@dependabot](https://github.com/dependabot))
+- Bump nock from 13.2.0 to 13.2.1 [#144](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/144) ([@dependabot](https://github.com/dependabot))
+- Bump eslint from 8.1.0 to 8.2.0 [#143](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/143) ([@dependabot](https://github.com/dependabot))
+- Bump nock from 13.1.4 to 13.2.0 [#142](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/142) ([@dependabot](https://github.com/dependabot))
+- Bump eslint-plugin-jest from 25.2.2 to 25.2.4 [#141](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/141) ([@dependabot](https://github.com/dependabot))
+- Bump eslint from 7.32.0 to 8.1.0 [#138](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/138) ([@dependabot](https://github.com/dependabot))
```

Comment:
Is it useful to have all these dependabot PRs in the changelog?

Reply:
I don't think dev dependency bumps is worth being mentioned, and if there are too many of the same prod bumps, its too noisy as well.

:+1: for at least removing all dev dependency stuff as a start and evaluating if there is further cleanup to do

Reply:
I've omitted irrelevant github action bumps as well from changelogs, but probably not consistently.